metrics/default_metrics/tarantool/init.lua

```diff
@@ -1,23 +1,52 @@
 local metrics = require('metrics')
 
 local default_metrics = {
-    require('metrics.default_metrics.tarantool.network'),
-    require('metrics.default_metrics.tarantool.operations'),
-    require('metrics.default_metrics.tarantool.system'),
-    require('metrics.default_metrics.tarantool.replicas'),
-    require('metrics.default_metrics.tarantool.info'),
-    require('metrics.default_metrics.tarantool.slab'),
-    require('metrics.default_metrics.tarantool.runtime'),
-    require('metrics.default_metrics.tarantool.memory'),
-    require('metrics.default_metrics.tarantool.spaces'),
-    require('metrics.default_metrics.tarantool.fibers'),
-    require('metrics.default_metrics.tarantool.cpu'),
-    require('metrics.tarantool.vinyl')
+    network = require('metrics.default_metrics.tarantool.network'),
+    operations = require('metrics.default_metrics.tarantool.operations'),
+    system = require('metrics.default_metrics.tarantool.system'),
+    replicas = require('metrics.default_metrics.tarantool.replicas'),
+    info = require('metrics.default_metrics.tarantool.info'),
+    slab = require('metrics.default_metrics.tarantool.slab'),
+    runtime = require('metrics.default_metrics.tarantool.runtime'),
+    memory = require('metrics.default_metrics.tarantool.memory'),
+    spaces = require('metrics.default_metrics.tarantool.spaces'),
+    fibers = require('metrics.default_metrics.tarantool.fibers'),
+    cpu = require('metrics.default_metrics.tarantool.cpu'),
+    vinyl = require('metrics.tarantool.vinyl'),
+    luajit = require('metrics.tarantool.luajit'),
+    cartridge_issues = require('metrics.cartridge.issues'),
+    clock = require('metrics.cartridge.clock'),
 }
 
-local function enable()
-    for _, metric in ipairs(default_metrics) do
-        metrics.register_callback(metric.update)
+local function enable(include, exclude)
```

Comment:
Since one may use `metrics` without a role, shouldn't we validate "mixing include and exclude" case here too?